microshift.sh

```diff
@@ -74,7 +67,11 @@ function enable_repos {
 function download_microshift_rpm {
     local pkgDir=$1
     pushd ${pkgDir}
-    brew download-build --quiet --arch noarch --arch ${ARCH} ${MICROSHIFT_NVR}
+    if [ -n "${MICROSHIFT_NVR-}" ]; then
+        yum download --downloadonly microshift-${MICROSHIFT_NVR-}
+    else
+        yum download --downloadonly microshift
```

Comment:
this will download only the microshift rpm not other sub packages which are part of NVR and required.

Reply:
I dropped the second commit until I test this more extensively.